pkg/volume/vspherevolume/vsphere_volume_util.go

```diff
@@ -63,10 +64,13 @@ const (
 	IopsLimitCapabilityMin              = 0
 )
 
+// ErrProbeVolume is an error instance for failed scanning of attached volumes.
 var ErrProbeVolume = errors.New("Error scanning attached volumes")
 
+// VsphereDiskUtil is an empty struct for vSphere volume
```

Comment:
what does this mean? The type is clear from the type :) What is it used for?

Reply:
Should i replace it with:
VsphereDiskUtil is the method receiver for vSphere volume functions?.

Reply:
no. Why does this exist?

Reply:
I think it's to make the code similar to OOP style.

Reply:
So from my first impression it seems like we can drop this one (would require some refactoring). I would recommend open an issue for the refactoring and we refactor this in a separate PR.

Reply:
Sure, i will open one after this.